libp2p/Host.h

```diff
@@ -236,5 +231,71 @@ class Host: public Worker
 	bool m_accepting = false;
 };
 	
+/**
+ * @brief Key material and derived secrets for TCP peer connection.
+ */
+struct PeerSecrets
+{
+	friend struct PeerHandshake;
+protected:
+	Secret encryptK;
+	Secret macK;
+	h256 egressMac;
+	h256 ingressMac;
+	
+	bytes magicCipherAndMac;
+	bytes recvdMagicCipherAndMac;
+};
+
+struct PeerHandshake: public std::enable_shared_from_this<PeerHandshake>
```

Comment:
Serious violation of coding standards. Please change this to a class and prefix its members with `m_`.